cfpb/cfgov-refresh
# Remove empowerment page

Remove hard-coded empowerment page in preparation for release of Wagtail replacement page on 12/8

## Removals

- Remove empowerment template
- Remove urls.py entry for empowerment page

## Testing

1. Check out branch
2. Navigate to [http://localhost:8000/practitioner-resources/economically-vulnerable/](http://localhost:8000/practitioner-resources/economically-vulnerable/) and verify that page 404s


## Checklist
